app/src/util/tools.spec.ts

```diff
@@ -177,4 +178,18 @@ describe('tools', () => {
       ).toThrow()
     })
   })
+
+  describe('truncateStringInTheMiddle', () => {
+    it('should not change string with truncate positions great or equal than string length', () => {
+      expect(truncate('foobar', 4, 2)).toBe('foobar')
+      expect(truncate('foobar', 6, 6)).toBe('foobar')
+      expect(truncate('foo')).toBe('foo')
```

Comment:
I don't know if it makes sense to have default arguments for this method.

Reply:
> I don't know if it makes sense to have default arguments for this method.

Maybe, the last two arguments must be required ?

Reply:
I've removed the last two parameters default values to make them explicit on each call of the function